typescript/infra/src/agents/keys.ts

```diff
@@ -103,9 +95,14 @@ export class ReadOnlyCloudAgentKey extends BaseCloudAgentKey {
   ): ReadOnlyCloudAgentKey {
     const { environment, context, role, chainName, index } =
       parseKeyIdentifier(identifier);
+    // apparently importing `environments` config breaks everything
```

Comment:
@asaj Hoping you have an explination for why
`import { environments } from '../../config/environments';`
in `keys.ts` causes 

```
An unexpected error occurred:

TypeError: Class extends value undefined is not a constructor or null
    at Object.<anonymous> (/Users/mattie/workspace/hyperlane-monorepo/typescript/infra/src/agents/gcp.ts:36:34)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Module.m._compile (/Users/mattie/workspace/hyperlane-monorepo/node_modules/ts-node/src/index.ts:1597:23)
    at Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Object.require.extensions.<computed> [as .ts] (/Users/mattie/workspace/hyperlane-monorepo/node_modules/ts-node/src/index.ts:1600:12)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/mattie/workspace/hyperlane-monorepo/typescript/infra/src/agents/index.ts:22:1)
```

Reply:
I'm afraid I do not..

Reply:
No objections